mip/mip.c

```diff
@@ -901,6 +917,12 @@ void mip_init(struct mg_mgr *mgr, struct mip_if *ifp) {
     ifp->mgr = mgr;
     mgr->extraconnsize = sizeof(struct connstate);
     if (ifp->ip == 0) ifp->enable_dhcp_client = true;
+
+    // Randomise initial ephemeral port
+    uint16_t jitter;
+    mg_random(&jitter, sizeof(jitter));
+    ifp->eport = MIP_ETHEMERAL_PORT + (jitter % (0xffffu - MIP_ETHEMERAL_PORT));
+
```

Comment:
We should change ETHEMERAL to EPHEMERAL

Reply:
Yeap, this will work for the STMs but won't be very useful without some entropy on other platforms without an RNG